src/session.ts

```diff
@@ -718,12 +736,25 @@ export class SessionManager implements Middleware {
     private showSessionMenu() {
         let menuItems: SessionMenuItem[] = [];
 
+        const currentExePath = (this.powerShellExePath || "").toLowerCase();
```

Comment:
This is SUPER nitpicky (I'm sorry) but is there a reason this line comes before the `getAvailablePowerShellExes` call instead of closer to where it's first used in this function (aka between 743-744 after the `if` vs where it is now)?

Reply:
No worries.   I could move it below `availablePowerShellExes` but `availablePowerShellExes` is referenced before `currentExePath`.  And both are used outside the `if/else if` statement

Reply:
ahhh I see it below now. No need to change it.